.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "doc/themes/hugo-theme-techdoc"]
```

Comment:
Can we vendor this via go modules somehow?

Reply:
I guess that can be doable, but why ? This isn't necessary to build the stack.

Note: It is going to be removed once we have our own theme